LetsDoStuff.WebApi/Services/DTO/AcceptRejectRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LetsDoStuff.WebApi.Services.DTO
+{
+    public class AcceptRejectRequest
```

Comment:
ParticipationRequest?

Reply:
We already have the ParticipationRequest for ParticipationController and it has a different body. But maybe it would be better to rename it something like "ParticipantRequest"?

Reply:
so maybe something like ParticipationResolutionRequest?